.changeset/friendly-months-fetch.md

```diff
@@ -0,0 +1,5 @@
+---
+'@iroha2/crypto-target-web': patch
+---
+
+**fix**: move `@iroha2/crypto-core` to direct dependencies
```

Comment:
in another changeset file you have `fix!` -- is that important? I'm not familiar with changeset files, but I'd guess you should follow the same syntax in all files

Reply:
Changeset doesn't have any constraints on changesets' content. I use `!` with [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/) in mind. Exclamation mark stands for a breaking change.

Reply:
ok, makes sense